scripts/jei/crafting_hints.zs

```diff
@@ -167,3 +167,35 @@ scripts.jei.requious.add(<assembly:crafting_hints>, { [
   <exnihilocreatio:block_barrel0> | <exnihilocreatio:block_barrel1>, null, null, null, null,
   Bucket('milk'), <minecraft:brown_mushroom> | <minecraft:red_mushroom>,
 ]: [<entity:minecraft:slime>.asStack(), <minecraft:slime>] });
+
+// general mineral sampling
+add1to1(<immersiveengineering:metal_device1:7>, <immersiveengineering:coresample>);
+
+// mineral sampling via Portable Drill
+add1to1(<portabledrill:portable_drill>, <immersiveengineering:coresample>);
+
+// Fast Mineral Sampling, see scripts/do/mineral_sampling.zs
+addInsOutsCatl(
+  [
+    <tconstruct:hammer>.withTag({
```

Comment:
Isnt a tool created this way would looks broken on hover? Please attach the screenshot.